thesis-de/content-de.tex

```diff
@@ -360,6 +362,128 @@ \subsubsection{Durchführung der Evaluierung}
 \end{outline}
 \nl%
 
+\subsection{Schnittstelle - Datenbereitstellung}
+\label{subsec:schnittstelle_datenbereitstellung}
+Neben der herkömmlichen Methode zur Datenabfrage von Datenbank-Systemen über
+einen sogenannten Datenbanktreiber, existiert noch eine weitere Möglichkeit in
+Form von einer sogenannten ``API''. API bedeuter im ausgeschriebenen
+``application programming interface'' und ermöglicht eine standardisierte
+Anbindung von externen Softwaresystemen. Dabei ist es egal, in welcher
+Programmiersprache oder auf welchem Protokoll das zu anbindende System arbeitet.
+
+Die Verwendung von einer Schnittstelle dieser Art besitzt Nachteile, sowie
+Vorteile beziehungsweise verbirgt gewisse Gefahren in Hinsicht auf den späteren
+Einsatz im Produktivsystem. Eine dieser Gefahren ist das
+vernachlässigen der Dokumentation einer API. Dadurch, das während der
+Entwicklung einer API bereits definiert wird, welche Fähigkeiten und Methodiken
+die API besitzen soll, wäre eine nicht vorhandene oder unvollständige
+Dokumentation suboptimal. Darum sollte bei der Auswahl von dem genutzen API
```

Comment:
fixed in https://github.com/DCC-org/documents/pull/68/commits/163c9d035ece91d491ebed273b49ae7bb2028366